src/commons/Sidebar/index.jsx

```diff
@@ -52,13 +52,14 @@ const SidebarItems = () => (
     <SidebarItem to="/" title="Home" />
     <SidebarItem to="/register" title="Register" />
     <SidebarItem to="/slots" title="Slots" />
+    <SidebarItem to="/leaderboard/public" title="Leaderboard" />
     <SidebarSeparator />
     <SidebarItem to="/users" title="Users" />
     <SidebarItem to="/colleges" title="Colleges" />
     <SidebarItem to="/events" title="Events" />
     <SidebarItem to="/judges" title="Judges" />
-    <SidebarSeparator />
     <SidebarItem to="/leaderboard" title="Leaderboard" />
```

Comment:
Dont you think public leaderboard should come here?

Reply:
At this point I don't care which is where. It's messy anyway. I'll rearrange everything once everything is done.